mailbox/event/event-rabbitmq/pom.xml

```diff
@@ -116,5 +116,14 @@
         </dependency>
     </dependencies>
 
-
+    <repositories>
+        <repository>
+            <id>spring-milestones</id>
```

Comment:
we won't merge that into master

Reply:
for sure, I only made the POC base on RC version of reactor-rabbitmq lib. Wait this library official release to make the change.